app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,24 @@
+class PostsController < ApplicationController
+  def show
+    @post = Post.find(params[:id])
+  end
+
+  def new
+  end
+
+  def create
+    @post = Post.new(post_params)
+ 
+    if @post.save
+      redirect_to @post
```

Comment:
This is where I think you should redirect to the index action instead.

Reply:
We agreed on that we as users would rather like to be able to have a quick "review" on our posts after its been created, since its easy to overlook a typo or grammatical issue when writing in a form, but once you've submitted you notice it